.github/dependabot.yml

```
@@ -1,12 +1,12 @@
-# package ecosystems to update and where the package manifests are located.
-# Please see the documentation for more information:
-# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
-# https://containers.dev/guide/dependabot
-
-version: 2
-updates:
- - package-ecosystem: "devcontainers"
-   directory: "/"
-   schedule:
-     interval: weekly
+# To get started with Dependabot version updates, you'll need to specify which
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)